mapie/control_risk/crc_rcps.py

```diff
@@ -0,0 +1,209 @@
+import numpy as np
+from numpy.typing import NDArray
+from typing import Optional, Tuple
+
+
+def get_r_hat_plus(
+    risks: NDArray,
+    lambdas: NDArray,
+    method: Optional[str],
+    bound: Optional[str],
+    delta: Optional[float],
+    sigma_init: Optional[float]
+) -> Tuple[NDArray, NDArray]:
+    """
+    Compute the upper bound of the loss for each lambda.
```

Comment:
UPDATE: could you add a short description telling to the user that this function follows the procedure of CRC or RCPS (then make ref to [1] and [2])?